bitnami/rabbitmq/values.yaml

```diff
@@ -498,7 +511,18 @@ nodeAffinityPreset:
 ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
 ## Note: podAffinityPreset, podAntiAffinityPreset, and  nodeAffinityPreset will be ignored when it's set
 ##
-affinity: {}
+affinity: |
+  podAntiAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchLabels: {{- include "common.labels.matchLabels" . | nindent 12 }}
+        topologyKey: kubernetes.io/hostname
+    preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          labelSelector:
+            matchLabels: {{- include "common.labels.matchLabels" . | nindent 14 }}
+          topologyKey: failure-domain.beta.kubernetes.io/zone
```

Comment:
That is the old label. It certainly still works, but deprecated for a long time, so not very future-proof.
cf: https://blablacar.atlassian.net/browse/DBRE-2411

Have you considered using the new one directly ?